cmd/create/cluster/cmd.go

```diff
@@ -493,37 +517,115 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	// Compute nodes:
-	computeNodes := args.computeNodes
-	// Compute node requirements for multi-AZ clusters are higher
-	if multiAZ && !cmd.Flags().Changed("compute-nodes") {
-		computeNodes = 3
-	}
-	if interactive.Enabled() {
-		computeNodes, err = interactive.GetInt(interactive.Input{
-			Question: "Compute nodes",
-			Help:     cmd.Flags().Lookup("compute-nodes").Usage,
-			Default:  computeNodes,
+	// Autoscaling
+	isAutoscalingSet := cmd.Flags().Changed("enable-autoscaling")
+	autoscaling := args.autoscalingEnabled
+	if !autoscaling && !isAutoscalingSet && interactive.Enabled() {
+		autoscaling, err = interactive.GetBool(interactive.Input{
+			Question: "Enable autoscaling",
+			Help:     cmd.Flags().Lookup("enable-autoscaling").Usage,
+			Default:  autoscaling,
+			Required: false,
 		})
 		if err != nil {
-			reporter.Errorf("Expected a valid number of compute nodes: %s", err)
+			reporter.Errorf("Expected a valid value for enable-autoscaling: %s", err)
 			os.Exit(1)
 		}
 	}
-	if multiAZ {
-		if computeNodes < 3 {
-			reporter.Errorf("The number of compute nodes needs to be at least 3")
+
+	isMinReplicasSet := cmd.Flags().Changed("min-replicas")
+	isMaxReplicasSet := cmd.Flags().Changed("max-replicas")
+
+	minReplicas := args.minReplicas
+	maxReplicas := args.maxReplicas
+	if autoscaling {
+
```

Comment:
The linter doesn't like this newline here...

Reply:
fixed